app/assess/routes.py

```diff
@@ -355,7 +367,9 @@ def qa_complete(application_id):
         sub_criteria=sub_criteria_banner_state,
         form=form,
         referrer=request.referrer,
-        display_status=sub_criteria_banner_state.workflow_status,
+        assessment_status=assessment_statuses[
+            sub_criteria_banner_state.workflow_status
+        ],
```

Comment:
Should this maybe use the `determine_assessment_status` function? Or because it's QA complete we don't need to?